src/api/Controllers/LithologicalDescriptionController.cs

```diff
@@ -0,0 +1,58 @@
+﻿using BDMS.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace BDMS.Controllers;
+
+[ApiController]
+[Route("api/v{version:apiVersion}/[controller]")]
+public class LithologicalDescriptionController : BdmsControllerBase<LithologicalDescription>
+{
+    private readonly BdmsContext context;
+
+    public LithologicalDescriptionController(BdmsContext context, ILogger<LithologicalDescription> logger)
+        : base(context, logger)
+    {
+        this.context = context;
+    }
+
+    /// <summary>
+    /// Asynchronously gets the <see cref="LithologicalDescription"/>s, optionally filtered by <paramref name="stratigraphyId"/>.
+    /// </summary>
+    /// <param name="stratigraphyId">The id of the stratigraphy referenced in the lithological descriptions to get.</param>
+    [HttpGet]
+    public async Task<ActionResult<IEnumerable<LithologicalDescription>>> GetAsync([FromQuery] int? stratigraphyId = null)
```

Comment:
PP: Da wir in dieser Methode immer nur Rückgabewerte vom Typ `IEnumerable<LithologicalDescription>` zurückgeben, kann hier direkt `Task<IEnumerable<LithologicalDescription>>` verwendet werden und auf `ActionResult` verzichtet werden.